### DISCUSSION TITLE
DNS_PROBE_FINISHED_NXDOMAIN

### DISCUSSION THREAD

**Discussion by stupidbrain** [mozilla/hubs#3489] [Hubs Cloud Questions] [answered]
My hub run normally yesterday, but showed 'DNS_PROBE_FINISHED_NXDOMAIN' today!  What is going on?  How do I fix it?

[Moderator deleted attached photo because of privacy and it's not needed for solving this error]

**Comment by robinkwilson** [comment]
Did you delete any CNAMES or A types on your domain?
Did you delete something in the AWS console that might've been a dependency?

If not, I'd do a quick web search for fixing DNS_PROBE_FINISHED_NXDOMAIN since it generally relates to your connection.
Look up `DNS_PROBE_FINISHED_NXDOMAIN` and you'll find some fixes.

**Comment by stupidbrain** [comment, marked as answer]
Dear CS,

Thank you for your help.  The problem is fixed when I restarted the stack.

I am deploying cloud hub in Hong Kong Education marketing, any resources
suggested?

I see there is a classroom on discord, may I implement that scene into my
hub?

Moses

在 2020年12月9日 週三 11:51，Robin K Wilson <notifications@github.com> 寫道：

**Reply by robinkwilson** [reply]
I think that's the classroom scene by one of our artists, if that is true, then absolutely use it! 

I'd deploy the Hubs Cloud in the Tokyo region. 

Good luck!
